spark.md

```diff
@@ -10,4 +10,9 @@ To get shell of Spark cluster run:
 spark-shell --master spark://<spark-master>:7077
 ```
 
-
+##SparkML
+The installation followed the instructions for [SparkML using Spark 2.1.1](http://spark.apache.org/docs/latest/ml-guide.html).
+All the dependencies should be installed. However, since Spark is a **pre-built tarball** the user should be aware of the [netlib-java dependency](http://spark.apache.org/docs/latest/ml-guide.html#dependencies) which requires an include.
```

Comment:
Check this file:
https://github.com/nlesc-sherlock/emma/blob/sparkML/roles/spark/tasks/main.yml#L167-L180

They are installed. Or did you mean something else?